src/main/scripts/org/reaktivity/specification/nukleus/mqtt/streams/connect.as.receiver.then.close/client.rpt

```diff
@@ -21,15 +21,15 @@ connect await ROUTED_CLIENT
 
 write nukleus:begin.ext ${mqtt:beginEx()
                              .typeId(nukleus:id("mqtt"))
-                             .role("RECEIVER")
+                             .capabilities("SUBSCRIBE")
                              .clientId("client")
                              .topic("sensor/one")
                              .subscriptionId(1)
                              .build()}
 
 read nukleus:begin.ext ${mqtt:beginEx()
                              .typeId(nukleus:id("mqtt"))
-                             .role("SENDER")
+                             .capabilities("PUBLISH")
```

Comment:
This should be `SUBSCRIBE` to confirm the capabilities of the bidirectional stream, i.e. that the initiator can publish.